tests/src/common.ts

```diff
@@ -51,3 +52,29 @@ export const installation2agent: InstallAgentsHapps = [
   [[chatDna]],
   [[chatDna]],
 ]
+
+// this mem_proof is a signature of the holo_agent key signing its public key
```

Comment:
"signing its" Does this mean that the memproof is the holo_agent signing it's own public key?  I though it should be signing the agent trying to join's public key.  But maybe it's only that for the test?

Reply:
Sorry that is and old comment. 
This is the payload it will e signing
```
 {
    role: String,
    record_locator: String
}
```